### PR TITLE
docs: editing doc of edit dataframe in "Sheet Management"

### PR DESCRIPTION
**Pull request by AChenQ**

### CODE REVIEW

**Comment by GravitiRob** [comment]
Task linked: [CU-2mmmtrb 添加更新数据功能的文档](https://app.clickup.com/t/2mmmtrb)

File: docs/source/features/sheet_management/dataframe.rst [hunk 1]
```diff
@@ -96,6 +96,36 @@ Read a DataFrame cell:
...
+
+   df.loc[0] = {"filename": "d.jpg"}
+
+Edit multiple rows.
+
+.. code:: python
+
+   df.loc[0:1] = [{"filename": "d.jpg"}, {"filename": "e.jpg"}]
+
+Edit The Items Of Column
```
Comment (graczhual) [inline docs/source/features/sheet_management/dataframe.rst:114]: the of should be lowercase

File: docs/source/features/sheet_management/dataframe.rst [hunk 2]
```diff
@@ -96,6 +96,36 @@ Read a DataFrame cell:
  Edit the DataFrame
 ********************
 
+Edit Rows
+=========
+
+Edit one row.
```
Comment (graczhual) [inline docs/source/features/sheet_management/dataframe.rst:102]: use `:`

File: docs/source/features/sheet_management/dataframe.rst [hunk 3]
```diff
@@ -96,6 +96,36 @@ Read a DataFrame cell:
...
+
+.. code:: python
+
+   df.loc[0] = {"filename": "d.jpg"}
```
Comment (graczhual) [inline docs/source/features/sheet_management/dataframe.rst:106]: do we need to write iloc here?

File: docs/source/features/sheet_management/dataframe.rst [hunk 4]
```diff
@@ -96,6 +96,36 @@ Read a DataFrame cell:
...
+
+.. code:: python
+
+   df["filename"][0] = "d.jpg"
+
+Edit multiple items.
+
+.. code:: python
+
+   df["filename"][0:1] = ["d.jpg", "e.jpg"]
```
Comment (graczhual) [inline docs/source/features/sheet_management/dataframe.rst:127]: do we need to add some examples like File, DataFrame?

File: docs/source/features/sheet_management/dataframe.rst (hunk 3 above)
Comment (AChenQ) [reply docs/source/features/sheet_management/dataframe.rst:106]: In this case, `loc` is the same as `iloc`.

File: docs/source/features/sheet_management/dataframe.rst (hunk 4 above)
Comment (AChenQ) [reply docs/source/features/sheet_management/dataframe.rst]: we introduce them in thier own section.

**Review by graczhual — APPROVED** [review]

**Pull request merged**